README.md

```diff
@@ -42,6 +42,7 @@ cd <project root>
 python3 -m venv .venv         # Create a Python virtual env
 source ./.venv/bin/activate   # Activate the virtual env for bash by source.
 pip install conan             # Install latest conan (conan=>2.0)
+pip install ninja             # Install ninja for multi-config build
```

Comment:
Nice, I did not realised you could install Ninja via pip.  Actually, to avoid multiple `pip install ...` statements can change this to use the project requirements.txt and add this to the requirements.
```suggestion
```